src/components/filter/filter.tsx

```diff
@@ -25,12 +33,6 @@ export const Filter: React.FC = () => {
 	const onToggleMyAdoptedTrees = () => {
 		if (!isLoggedIn()) {
 			setIsTooltipVisible(!isTooltipVisible);
```

Comment:
I think, since in the useEffect we go only "one way" to set `isTooltipVisible` to `false` we should call it here also "one way" by only setting it to `true`. Not that we accidentally set it to false by clicking again.